static/images/landing-page/research/interactive_messaging.json

```diff
@@ -0,0 +1,90 @@
+                }
+            },
+            {
+                "sender": "Zoe Davis",
+                "content": "The fastest known algorithm is the general number field sieve, which heuristically runs in time:\n$$\\exp\\left(\\left(\\sqrt[3]{\\frac{64}{9}} + o(1)\\right) + (\\ln n)^{\\frac{1}{3}} (\\ln\\ln n)^{\\frac{2}{3}}\\right)$$\n However, it is much more complicated.",
+                "reactions": {
+                    "runner": ["Manvir Singh"],
+                    "+1": ["Manvir Singh", "Iago"]
+                }
+            }
+        ],
+        "screenshot": "static/images/landing-page/research/image_1.png"
+    },
+    {
+        "recipient_type": "channel",
+        "channel": "general",
+        "color": "#9987E1",
+        "topic": "paper posted",
+        "users": {
+            "John Lin": "static/images/characters/starr.png",
+            "Anna Smith": "static/images/characters/starr.png",
+            "John Doe": ""
+        },
+        "messages": [
+            {
+                "sender": "John Lin",
+                "content": "Hi all! We just posted a pre-publication draft of our paper! Sharing here for feedback, suggestions, etc. [paper.pdf](#)",
+                "reactions": {
+                    "tada": ["John Lin", "Iago", "Anna Smith", "John Doe"],
+                    "eyes": ["Anna Smith", "Iago"]
+                }
+            },
+            {
+                "sender": "Anna Smith",
+                "content": "Just noticed a small typo in Figure 4. The caption says `detales` instead of `details`. :smile:",
+                "reactions": {
+                    "check": ["John Lin"]
+                }
+            }
+        ],
+        "screenshot": "static/images/landing-page/research/image_2.png"
+    },
+    {
+        "recipient_type": "channel",
+        "channel": "thesis support group",
+        "color": "#95A5FD",
+        "topic": "chat live",
+        "users": {
+            "Zoe Davis": "static/images/characters/starr.png",
+            "James Williams": "static/images/characters/starr.png",
+            "John Doe": "",
+            "Abraham Lincoln": ""
+        },
+        "messages": [
+            {
+                "sender": "Zoe Davis",
+                "content": "Hey! How does <time:2024-05-10T17:00:00+05:30> sound for doing a video call today? My paper just got rejected.",
+                "reactions": {
+                    "+1": ["James Williams", "Iago", "Zoe Davis", "John Doe", "Abraham Lincoln"],
+                    "broken_heart": ["James Williams", "Iago", "Zoe Davis", "Abraham Lincoln"]
+                }
+            },
+            {
+                "sender": "James Williams",
+                "content": "I need to send off a draft of chapter 2 to my advisor. Will try to make it!"
+            },
+            {
+                "sender": "Zoe Davis",
+                "content": "@**everyone** Video call is happening now! [Click to join video call](#)"
```

Comment:
Let's change [Click to join video call] -> [Join video call.], since we changed the default text the button inserts.

Reply:
updated!